docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1059,7 +1058,30 @@ BESU_PRIVACY_URL=http://127.0.0.1:8888
 privacy-url="http://127.0.0.1:8888"
 ```
 
-URL on which the [Orion node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is running.    
+URL on which the [Orion node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is running.
+
+### pruning-enabled
+
+```bash tab="Syntax"
+--pruning-enabled
+```
+
+```bash tab="Command Line"
+--pruning-enabled=true
+```
+
+```bash tab="Environment Variable"
+BESU_PRUNING_ENABLED=true
+```
+
+```bash tab="Configuration File"
+pruning-enabled=true
+```
+
+Enables [pruning](../../Concepts/Pruning.md) to reduce storage required for the world state size.
```

Comment:
I think we should remove `size` here. We're technically not worried about storing the size, we're worried about storing the thing itself :)

Reply:
I believe the phrase came from a different comment during review. In any case, I removed the term.